app/build.gradle

```diff
@@ -37,8 +38,8 @@ android {
 
 dependencies {
 
-    implementation 'androidx.core:core-ktx:1.8.0'
-    implementation 'androidx.appcompat:appcompat:1.5.0'
+    implementation 'androidx.core:core-ktx:1.9.0'
+    implementation 'androidx.appcompat:appcompat:1.5.1'
```

Comment:
❗  I believe with the changes here they require bumps of `targetSdk` and `compileSdk` to API 33. A revert of these version changes should work, or bumping target and compile SDKs (above in the `android` block at the top of this file) to 33 also resolves the issue I was seeing.